data/hai/hai news.txt

```diff
@@ -0,0 +1,251 @@
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+# Visual suggestion: a bit quirky or weird; expression is content, confident, or stoic
+news "food in Hai space"
+	location
+		government "Hai"
+	name
+		word
+			"Visiting chef"
+			"Food connoisseur"
+			"Tourist"
+			"Tour guide"
+	portrait
+		portrait/human26
+		portrait/human31
+		portrait/human57
+		portrait/human68
+	message
+		word
+			"This may sound odd, but the acorn cookies they sell here are the best cookies I've ever eaten."
+			"A Hai-Human fusion restaurant near here sells peanut pizza. It sounds gross, but the taste is heavenly."
+			"They make beer from fermented nuts here. I didn't know you could do that!"
+			"I just can't get enough acorn stew. These Hai cooks are amazing."
+			"Peanut this, walnut that, hazelnut icecream. What I wouldn't give for some good steak right now..."
+
+
+# Visual suggestion: glasses, clean-cut attire, possibly scientific accoutrements
+news "trading in Hai space"
+	location
+		government "Hai"
+	name
+		word
+			"Merchant captain"
+	portrait
+		portrait/human02
+		portrait/human04
+		portrait/human06
+		portrait/human10
+		portrait/human15
+		portrait/human17
+		portrait/human31
+		portrait/human58
+		portrait/human63
+		portrait/human72
+		portrait/human73
+		portrait/human76
+		portrait/human79
+		portrait/human98
+		portrait/human108
+		portrait/human112
+		portrait/human125
+	message
+		word
+			"Hai Tech is incredible compared to ours. Every time I add a ship to my fleet, I always make a trip up here to stock up on Hai tech."
+			"Have you tried out one of those Hai anti-missile systems? They'll waste any missile in Human space."
+			"Can you believe these Hai? They're light-years ahead of us, right under our noses."
+			"Have you visited the Quarg ringworld here in Hai space? It's a humbling sight. There's probably more Quarg in there than there are humans anywhere. Makes you feel pretty insignificant."
+			"Did you see the stuff they sell on Icelake? Hai technology is amazing."
+			"I met some young Hai pilots. It sounds like they start out as poor as we do."
+			"I sold my entire stock of food at this port. I guess even Hai get tired of nuts."
+			"Some Hai were protesting the work week. They want to work only 32 hours a week instead of 40. I used to work 70 hours a week for the Syndicate!"
+
+
+# Visual suggestion: glasses, clean-cut attire, possibly scientific accoutrements
+news "science in Hai space"
+	location
+		government "Hai"
+	name
+		word
+			"Scientist"
+			"Engineer"
+	portrait
+		portrait/human13
+		portrait/human37
+		portrait/human43
+		portrait/human64
+		portrait/human71
+		portrait/human83
+	message
+		word
+			"A while back, my Hai research partner was nearing a breakthrough, and then she vanished. I haven't seen her since. She probably joined the Unfettered."
```

Comment:
Do we really want to outright state "She probably joined the Unfettered?" This idea seems like one of those things that the news should only hint at, not explicitly state, with spaceport mission being where we really tackle the idea.

Reply:
She didn't join the unfettered. She was kidnapped by the Devil-Run pirates. The purpose of this and some other new spaceport news is to set up the lore for the Devil-Run kidnappings. I removed the reference to the unfettered.